docs-requirements.in

```diff
@@ -0,0 +1,9 @@
+https://github.com/release-engineering/estuary-api/tarball/master#egg=estuary
```

Comment:
Could you please only add the dependencies that are specific to the docs since when you run `pip-compile`, it is also using the dependencies in `setup.py`?